doc/household.md

```diff
@@ -0,0 +1,166 @@
+> of the vertical line making the company’s general ledger easier to read.
+>
+
+Note that
+in double-entry accounting,
+a credit can both increase and 
+decrease an account balance.
+For example, 
+when you credit your savings account
+(an asset)
+the balance goes down!
+I have to use
+the table in
+the next section (Rule #3)
+to consistently 
+get the T-charts right.
+
+### Rule #3: The five different types of accounts.
+
+| Account     | Increased By | Decreased By | Examples                                     |
+|-------------|--------------|--------------|----------------------------------------------|
+| Assets      | Debit        | Credit       | cash, checking account, real estate          |
+| Expenses    | Debit        | Credit       | food, entertainment, interest payments, etc. |
+| Liabilities | Credit       | Debit        | mortgage principle, credit card balance      |
+| Equity      | Credit       | Debit        | your net worth                               |
+| Revenue     | Credit       | Debit        | paycheck, interest earned, bond dividends    |
+
+
+
+## Example Transactions
+
+### Example #1: A paycheck is deposited in your checking account.
+
+```
+ Debit                | Credit
+----------------------+--------------
+checking account      | income
+```
+
+* Your checking account balance increases.  A checking account 
+  is an asset, 
+  so read 
+  the first row 
+  in the rule #3 table.  To increase an asset account
+  you debit it.
+* Your paycheck income goes up.  Income is revenue,
+  so read
+  the fifth row
+  in the rule #3 table. To increase a revenue account
+  you credit it.
+
+
+### Example #2: Your pay your monthly Netflix bill.
+```
+ Debit                | Credit
+----------------------+--------------
+ media expense        | checking account
+```
+
+
+* Your checking account balance decreases.
+* Your media expense balance increases.
+
+### Example #3: You pay your mortgage.
+```
+ Debit                | Credit
+----------------------+--------------
+ interest expense     | checking account
+ principle still due  | checking account
+```
+
+* Interest on the mortgage is an expense, and it increases.
+* The principle left on your mortgage is a liability, and it decreases.
+
+
+### Example #4: The value of your house goes up.
+```
+ Debit                | Credit
+----------------------+--------------
+ house value          | equity
+```
+
+Businesses don't need 
+to track this change
+as it is an unrealized gain.
+
+But a household may
+want to track 
+their current net worth.
+
+
+### Example #5: Your 401k earns interest and dividends.
+```
+ Debit                | Credit
+----------------------+--------------
+ retirement accounts  | equity
+```
+
+See Example #4.
+
+### Example #6: You want track taxes withheld from your paycheck an expense.
+
+
+```
+ Debit                | Credit
+----------------------+--------------
+ Federal tax expense  | Paycheck income
+```
+
+This one is tricky.
```

Comment:
Any reason for the narrow markdown formatting? The markdown style I see most often follows these rules:

1. Always insert a line break after each sentence (e. g. after a question mark or full stop).
2. No spaces at the end of a sentence (e. g. between a question mark and the following line break).
3. Insert additional line breaks as necessary if a single sentence gets too wide.

For example:
```
This one is tricky.
Why does income (a revenue account) also increase?
It is because your gross income includes your tax withholding.
```

Reply:
A new line between clauses matches how I think about the topic.  I find it makes it easier to edit.  Unfortunately, markdown renderer is not a great match for this style, as the rendered newlines depend on if the source newline is between words or after a sentance end.